backend/src/v5/middleware/auth.js

```diff
@@ -24,7 +25,25 @@ const { validateMany } = require('./common');
 
 const AuthMiddleware = {};
 
-AuthMiddleware.validSession = async (req, res, next) => {
+const isIpAddressValid = async (req, res, next) => {
+	const ipMatch = req.session.ipAddress === (req.ips[0] || req.ip);
+
+	if (req.session.user.isAPIKey || ipMatch) {
+		await next();
+		return;
+	}
+
+	try {
+		req.session.destroy(() => {
+			res.clearCookie('connect.sid', { domain: config.cookie_domain, path: '/' });
```

Comment:
`connect.sid` is a constant in `utils/sessions.constant.js`, so we should reference that

Reply:
probably should clear the CSRF token too